tests/migration/migration.go

```diff
@@ -1615,10 +1615,10 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 					By("checking that we were never able to connect")
 					tlsErrorFound := false
 					for err := range errors {
-						if strings.Contains(err.Error(), "remote error: tls: bad certificate") {
+						if strings.Contains(err.Error(), "remote error: tls: unknown certificate authority") {
```

Comment:
This has to change in go implementation. It would be great to reference the commit.

Reply:
Added this information to the commit - https://github.com/kubevirt/kubevirt/pull/10914/commits/5cd924bf8df35b8e3bc0e9ea35d724349838c389